dev_tools/scripts/create_new_algorithm.sh

```diff
@@ -97,11 +97,13 @@ TEMPLATE_FILES=(
     "${name}_kernels.hpp"
     "${name}_kernels.cpp"
     "${name}_kernels.cpp"
-    "${name}_kernels.c*"
+    "${name}*.[ch]*"
```

Comment:
Is there a reason for that change for the CUDA kernels? If I use `cg2` as the new name and base it on `solver cg`, it creates both `cg2_kernels.cu` and `cg2s_kernels.cu`.
As far as I can tell, this currently only happens with `cg`, since `cgs` has `cg` as a prefix.

Reply:
Good point. The change is for jacobi though as this has multiple files and all need to be matched (including a header). I will put a `_` there which restrict the sort of names you can give, but indeed removes the bug that you talk about.

Reply:
I think that is fine since the name should not really contain an `_`.